cfme/tests/test_rest.py

```diff
@@ -26,7 +26,7 @@
     scope="module"
 )
 
-pytestmark = [pytest.mark.ignore_stream("5.2")]
+pytestmark = [pytest.mark.ignore_stream("5.3")]
```

Comment:
I think we have already removed support for 5_3, and we tagged a while ago where the last known support for 5_3 was